<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dtuinfomationDialog</class>
 <widget class="QDialog" name="dtuinfomationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page1">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QLabel" name="lab1">
         <property name="text">
          <string>硬件遥信</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTableWidget" name="tableWidget1"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="lab2">
         <property name="text">
          <string>遥测值</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QTableWidget" name="tableWidget2"/>
       </item>
       <item row="0" column="0" rowspan="2">
        <widget class="dtuarminfoWidget" name="widget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page2">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget3"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget4"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>dtuarminfoWidget</class>
   <extends>QWidget</extends>
   <header location="global">dtuarminfowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
